src/common.h

```diff
@@ -110,6 +110,8 @@ struct testEngine {
 extern struct testEngine ncclTestEngine;
 
 struct threadArgs {
+  char setup_file[64];
```

Comment:
same question with 64, OR, even does it need an array or a const char pointer would do depending on who is holding the memory